decidim-proposals/spec/lib/decidim/proposals/component_spec.rb

```diff
@@ -148,6 +148,32 @@
       end
     end
 
+    context "when support limit per participant is empty" do
+      before do
+        visit edit_component_path
+        component.update(settings: { vote_limit: "" })
+        visit edit_component_path
+      end
+
+      it "does NOT allow updating proposal component" do
```

Comment:
As we're talking about consistency on the use of the 'NOT' language in other PRs, I think it'd be better to use the lowercase variant:
```suggestion
      it "does not allow updating proposal component" do
```
The reasoning is that it's more used in decidim specs: 

```bash
$ grep -r "does not" decidim*/spec/ |wc -l
1016
$ grep -r "does NOT" decidim*/spec/ |wc -l
15
```

We should probably change those and also add it to the spelling CI PR (#10553)